testnet/stacks-node/src/burnchains/bitcoin_regtest_controller.rs

```diff
@@ -663,8 +676,11 @@ impl BitcoinRegtestController {
         total_spent: u64,
         utxos: Vec<UTXO>,
         signer: &mut BurnchainOpSigner,
+        attempt: u64,
     ) -> Option<()> {
-        let tx_fee = self.config.burnchain.burnchain_op_tx_fee;
+        // TODO: crude RBF -- total tx fee and tx fee per vbyte must both be greater than they were
+        // before.
+        let tx_fee = self.config.burnchain.burnchain_op_tx_fee + attempt * DUST_UTXO_LIMIT;
```

Comment:
if I understand correctly how RBF works, this may be overpaying, i.e. at least the relay fee for the message size, which is set at set at 1000 sats per kB and the commit message is less than 400 bytes.  Or perhaps this should be configurable.  We should just file a separate issue as not to hold this PRS

Reply:
Yeah, you're right.  It's "crude RBF" ;)  The code here should discover the remote bitcoind's minimum relay fee and remember how long the last transaction was so it can get a tighter estimate for how many extra satoshis need to be added to the fee.